src/main/java/com/ldtteam/structurize/storage/ClientStructurePackLoader.java

```diff
@@ -179,18 +179,6 @@ else if (Minecraft.getInstance().level == null && (loadingState == ClientLoading
     public static void onServerSyncAttempt(final Map<String, Double> serverStructurePacks)
     {
         Network.getNetwork().sendToServer(new SyncSettingsToServer());
-
-        if (serverStructurePacks.isEmpty())
```

Comment:
We should keep the early return and rather extract the disable logic into a method and call it on early return as well. Otherwise we run into sync issues.